packages/react-core/src/components/Page/examples/PageWithOrWithoutFill.tsx

```diff
@@ -60,13 +60,12 @@ export const PageWithOrWithoutFill: React.FunctionComponent = () => {
   );
 
   return (
-    <Page masthead={masthead} sidebar={sidebar}>
+    <Page isContentFilled masthead={masthead} sidebar={sidebar}>
       <PageSection>A default page section</PageSection>
-      <PageSection isFilled={true}>This section fills the available space.</PageSection>
-      <PageSection isFilled={false}>
-        This section is set to not fill the available space, since the last page section is set to fill the available
-        space by default.
+      <PageSection isFilled={true} variant="secondary">
+        This section fills the available space.
       </PageSection>
+      <PageSection isFilled={false}>This section is set to not fill the available space.</PageSection>
```

Comment:
This demo set `isFilled={false}` in v5 since the last section would fill by default, but that's no longer needed for v6 so I'd take this out so it isn't confusing.

```suggestion
      <PageSection>A default page section</PageSection>
```